lib/generators/react_on_rails/generator_helper.rb

```diff
@@ -17,6 +17,14 @@ def puts_setup_file_error(file, data)
     puts "\n#{data}\n"
   end
 
+  def return_setup_file_error(file, data)
+    error = ""
+    error << "** #{file} was not found.\n"
+    error << "Please add the following content to your #{file} file:\n"
+    error << "\n#{data}\n"
+    error
+  end
+
```

Comment:
Renamed the string to 'error' for clarity. 

Also removed the strip_heredoc method per @robwise comment on data already being left justified causing it to not work here.

Output looks like this now:
```
** .gitignore was not found.
Please add the following content to your .gitignore file:

# React on Rails
npm-debug.log
node_modules

# Generated js bundles
/app/assets/javascripts/generated/*
```